File: Serenity Practice/HosbitalSerenity.Web/Modules/ServerTypes/Default/CityColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { CityRow } from "./CityRow";

export interface CityColumns {
    CityId: Column<CityRow>;
    CityName: Column<CityRow>;
    GovName: Column<CityRow>;
}

export class CityColumns extends ColumnsBase<CityRow> {
    static readonly columnsKey = 'Default.City';
    static readonly Fields = fieldsProxy<CityColumns>();
}

File: Serenity Practice/HosbitalSerenity.Web/Modules/Default/City/CityColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace HosbitalSerenity.Default.Columns;

[ColumnsScript("Default.City")]
[BasedOnRow(typeof(CityRow), CheckNames = true)]
public class CityColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int CityId { get; set; }
    [EditLink]
    public string CityName { get; set; }
    public string GovName { get; set; }
}